src/components/search/__tests__/search.test.js

```diff
@@ -119,4 +119,20 @@ describe('search', () => {
       expect(tree).toMatchSnapshot();
     });
   });
+
+  describe(testCases.withConnector.description, () => {
+    let testCase;
+    let wrapper;
+    let tree;
+
+    beforeEach(() => {
+      testCase = testCases.withLayout;
+      wrapper = renderer.create(testCase.element);
+      tree = wrapper.toJSON();
+    });
+
+    test('renders as expected', () => {
+      expect(tree).toMatchSnapshot();
+    });
```

Comment:
@katydecorah any suggestions on how I might make some kind of assertion that the connector provided to the test case is actually in use? It shouldn't actually change anything about how the component renders, only how it behaves.

Reply:
Perhaps we could use https://jestjs.io/docs/en/expect#tobeinstanceofclass?

Something like:

```
expect(testCase.props.connector).toBeInstanceOf(SiteSearchAPIConnector);
```

Reply:
Added two expectations -- the instance of check and also a check on the connector's properties to make sure its not using the default values.